{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/factories.py

```diff
@@ -1,11 +1,12 @@
 from collections.abc import Sequence
 from typing import Any
 
-from django.contrib.auth import get_user_model
 from factory import Faker
 from factory import post_generation
```

Comment:
Ruff split this import into 2 lines, was this expected?

Reply:
We just switched to Ruff and we didn't really discuss that. Feel free to open an issue if you want to discuss it further, I don't have any strong opinion, only a slight habit of grouping the imports, but that's mostly following what isort does by default, I think. I guess the argument for putting them on separate line is that it makes diff clearer?